conf/logback-test.xml

```diff
@@ -34,14 +22,6 @@
         </encoder>
     </appender>
 
-    <logger name="accesslog" level="OFF" additivity="false">
-        <appender-ref ref="ACCESS_LOG_FILE" />
-    </logger>
-
-    <logger name="com.ning.http.client.providers.netty" additivity="false">
-        <appender-ref ref="STDOUT_IGNORE_NETTY" />
-    </logger>
-
     <logger name="com.google.inject" level="OFF"/>
 
     <logger name="org.asynchttpclient.netty" level="OFF"/>
```

Comment:
These entries relating to netty can also be removed:
logger name="org.asynchttpclient.netty" level="OFF" 
logger name="io.netty.buffer" level="OFF"/
logger name="play.core.netty" level="OFF"

Reply:
All - removed - resolved